src/composables/useDelegates.ts

```diff
@@ -142,6 +149,43 @@ export function useDelegates(delegatesConfig: DelegatesConfig) {
     return tx;
   }
 
+  async function fetchDelegateVotesAndProposals(
+    delegates: string[],
+    space: string
```

Comment:
```suggestion
    delegates: string[]
```

Reply:
Space is not used currently for demo purposes